lib/job-simulation.rb

```diff
@@ -2,19 +2,47 @@
 require './Queue.rb'
 
 class JobSimulation
-  attr_reader :workers, :waiting, :roll
+  #attr_reader :workers, :waiting, :roll
 
-  def initialize (jobs_available, job_seekers)
+  def initialize (job_seekers)
+    jobs_available = 6
```

Comment:
This cannot be assumed as now the functionality of our code has been limited. Before this we had the capability of saying there are 28 jobs and 100 people wanting jobs; with this change, there are always only 6 jobs available.